### ISSUE TITLE
[Bug]: Class prototype should not be writable

### ISSUE THREAD

**Issue by p51lee** [swc-project/swc#6357]
### Describe the bug

According to the [section 20.2.2.2 of ECMAScript specification](https://tc39.es/ecma262/2022/multipage/fundamental-objects.html#sec-function.prototype), a `prototype` of class `x` should not be writable:

<img width="953" alt="image" src="https://user-images.githubusercontent.com/68288688/200067779-c802bbfb-5ed9-4e47-aaf3-3f2e195c8c58.png">

### Input code

```typescript
class x { }
console.log(x.prototype);
x.prototype = 1;
console.log(x.prototype);
```

### Playground link

https://play.swc.rs/?version=1.3.14&code=H4sIAAAAAAAAA0vOSSwuVqhQqFao5UrOzyvOz0nVy8lP16jQKyjKL8kvqSxI1bTmQuIp2CoYWuNWCgDdi3dYUAAAAA%3D%3D&config=H4sIAAAAAAAAA0WMTQrEIAxG75K1286id5hDBCctFv9IUhgR714tlu7C915ehUMsrBUyshCPS0pU%2FMMKZAOKZZcVTNf6tKEXagYUeScdiiyd%2BZSEJjUQXHRbGSWbQmYSeRHG3T9m66GQfucYKmjJdAc%2F0N7G%2FHPynaLySe0Cj9ke9LUAAAA%3D

### Expected behavior

Class prototype is not writable so `x.prototype = 1` should not change `x.prototype`:
```console
$ node input.js
{}
{}
```

### Actual behavior

In `output.js`, `x.prototype` is changed to `1`:
```console
$ node output.js
{}
1
```

### Version

1.3.14 (default SWC playground)

**Comment by Austaras** [comment]
In babel this is fixed by babel/babel#12115 five years after someone fired an issue. I don't think this is of much value, and it would add a size penalty for every class -- however maybe it doesn't matter as there's not many people still shipping ES5.

**Comment by kdy1** [comment]
I think this is useless, thank you @Austaras
